rickchurch/constants.py

```diff
@@ -3,18 +3,24 @@
 from decouple import config
 
 
+DISCORD_BASE_URL = "https://discord.com/api"
+
 log_level: str = config("LOG_LEVEL", default="INFO")
 
 base_url: str = config("BASE_URL")  # URL to host church of rick
-discord_token_url: str = config("TOKEN_URL", default="https://discord.com/api/oauth2/token")
-discord_user_url: str = config("USER_URL", default="https://discord.com/api/users/@me")
 
 # Get these from https://discord.com/developers/applications, OAuth2 section
 client_id: str = config("CLIENT_ID")
 client_secret: str = config("CLIENT_SECRET")
 # Get this by adding {base_url}/oauth_callback as URI in the application's OAuth2 section
 oauth_redirect_url: str = config("OAUTH_REDIRECT_URL")
 
+enable_auto_join: bool = bool(config("ENABLE_DISCORD_AUTOJOIN", ""))
```

Comment:
`decouple.config` has a `cast` kwarg you can use that does conversion automatically, using that would make more sense. Also, I'd prefer using the `default` kwarg instead of just 2nd positional arg, since it makes it cleaner what that is, though in this case, you used `""` as default, which isn't a bool value, use `True` (or `False`) instead
```suggestion
enable_auto_join: bool = config("ENABLE_DISCORD_AUTOJOIN", default=True, cast=bool)
```